vault/tests/test_integration.py

```diff
@@ -42,8 +44,9 @@ def test_integration_noauth(aggregator, dd_run_check, check, no_token_instance,
 @auth_required
 @pytest.mark.e2e
 @pytest.mark.parametrize('use_openmetrics', [False, True], indirect=True)
-def test_e2e(dd_agent_check, e2e_instance, global_tags, use_openmetrics):
-    instance = dict(e2e_instance)
+@pytest.mark.parametrize('use_auth_file', [False, True])
```

Comment:
I would move this to a file test_e2e.py for more organization of tests

Reply:
Good call. Moved it!